src/enums/uploadFilesMethods.js

```diff
@@ -0,0 +1,8 @@
+const UPLOAD_FILES_METHODS = {
+  S3: 's3',
+  DEFAULT: 'default',
```

Comment:
Only default is needed, this file should probably not exist

Reply:
This const is used in the configuration of the FileDashboardUploader. IMO it is useful to have it in case we want to provide with other methods for uploading files.